.github/workflows/nfr.yml

```
@@ -0,0 +1,191 @@
+name: Non Functional Testing
+
+on:
+  workflow_dispatch:
+    inputs:
+      test_label:
+        description: NFR test to run. Choose between performance, upgrade or all.
```

```suggestion
        description: NFR test to run. Choose between performance, upgrade, or all
```
nit:
Maybe grammar difference is because of different style of english, but I think this conforms to American english 😆 . Without this its a little confusing for my eyes.

Also, conform with the other descriptions that don't have a period, or add period to the other descriptions.

---

Ah the Oxford comma causing controversy again 😉 

> Opinions among writers and editors differ on whether to use the serial comma, and usage also differs somewhat between regional varieties of English. British English allows constructions with or without this comma, whereas in American English the comma is common and even considered mandatory by some style guides. -> [Serial_comma](https://en.wikipedia.org/wiki/Serial_comma)

Of course, technically, I use Hiberno-English 🤣 That being said, I've added the comma 😆

---

Us Americans just want to feel more proper by using the prestigious Oxford comma